deployment/keystone/changeset/internal/state.go

```diff
@@ -123,3 +133,35 @@ func loadContractSet(lggr logger.Logger, chain deployment.Chain, addresses map[s
 	}
 	return &out, nil
 }
+
+// getOCR3Contract returns the OCR3 contract from the contract set.  By default, it returns the only
+// contract in the set if there is no address specified.  If an address is specified, it returns the
+// contract with that address.  If the address is specified but not found in the contract set, it returns
+// an error.
+func getOCR3Contract(contracts map[common.Address]*ocr3_capability.OCR3Capability, addr *common.Address) (*ocr3_capability.OCR3Capability, error) {
```

Comment:
@MStreet3 Style nit: IMO this feels a bit like premature optimization -- why not just inline this code into the ContractSet GetOCR3Contract method?